maxim/logger/gemini/utils.py

```
@@ -178,16 +180,20 @@ def parse_content(
                     message["role"] = content.role
             if content.parts is not None:
                 for part in content.parts:
-                    if part is not None and part.text is not None:
+                    if (
+                        part is not None
+                        and part.text is not None
+                        and isinstance(message["content"], str)
+                    ):
                         message["content"] += part.text
         elif isinstance(content, List):
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)